crates/edr_napi/src/trace/solidity_tracer.rs

```diff
@@ -0,0 +1,342 @@
+use napi::{
+    bindgen_prelude::{Either3, Either4},
+    Either, Env,
+};
+use napi_derive::napi;
+
+use super::{
+    error_inferrer::{
+        instruction_to_callstack_stack_trace_entry, ErrorInferrer, SubmessageDataRef,
+    },
+    mapped_inlined_internal_functions_heuristics::{
+        adjust_stack_trace, stack_trace_may_require_adjustments,
+    },
+    message_trace::{CallMessageTrace, CreateMessageTrace, EvmStep, PrecompileMessageTrace},
+    model::{Instruction, JumpType},
+    opcodes::Opcode,
+    solidity_stack_trace::{PrecompileErrorStackTraceEntry, SolidityStackTrace},
+};
+use crate::{
+    trace::{
+        exit::ExitCode,
+        solidity_stack_trace::{
+            ContractTooLargeErrorStackTraceEntry, SolidityStackTraceEntry,
+            StackTraceEntryTypeConst, UnrecognizedContractCallstackEntryStackTraceEntry,
+            UnrecognizedContractErrorStackTraceEntry,
+            UnrecognizedCreateCallstackEntryStackTraceEntry,
+            UnrecognizedCreateErrorStackTraceEntry,
+        },
+    },
+    utils::ClassInstanceRef,
+};
+
+#[napi(constructor)]
+pub struct SolidityTracer();
```

Comment:
```suggestion
pub struct SolidityTracer;
```

Unit struct

Reply:
Ah, I forgot to remove the braces after pruning some state. Thanks!